tests/simple_standard_scenarios_datasets.py

```diff
@@ -312,5 +319,66 @@ def perform_op_by_electricity(stats, table, name, operation):
     stats[name]["count"] = table.count()
 
 
+def make_unpivoted_datasets():
```

Comment:
This is a substantial behavioral change. After filtering the SS datasets the code will now unpivot the two comstock tables. That gives us two-table and one-table pivoted datasets that we can query in the tests with no changes to that test code.